hooks/applypatch-msg

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Base Git hook template from https://github.com/rycus86/githooks
+#
+# It allows you to have a .githooks folder per-project that contains
+# its hooks to execute on various Git triggers.
+
+# Read the runner script from the local/global or system config
+# Auto-generated: do not edit!
```

Comment:
/nit: `Read the runner script from the local/global or system config` is not actually happening here which is a bit confusing, but let's maybe look into it as another change.

Reply:
Will change that, later.

Reply:
Ah you merged already, lool, ok. I will make another knitpik PR for that one... :-)

Reply:
Yeah, since they are not used yet, we can deal with it when we do use them.